client/src/App.tsx

```diff
@@ -26,13 +27,14 @@ function App(): JSX.Element {
               <Route exact path="/login" component={Login} />
               <Route exact path="/signup" component={Signup} />
               <ProtectedRoute exact path="/fileUpload" component={DesignSubmit} />
+              <ProtectedRoute exact path="/setting" component={ProfileSetting} />
               {/* Replace component with Discovery Page */}
               <Route exact path="/" component={Dashboard} />
               {/* Replace the components once created */}
               <ProtectedRoute exact path="/messages" component={Dashboard} />
               <ProtectedRoute exact path="/notifications" component={Dashboard} />
               <ProtectedRoute exact path="/new-contest" component={Dashboard} />
-              <ProtectedRoute exact path="/profile" component={Dashboard} />
+              <ProtectedRoute exact path="/profile" component={ProfileSetting} />
```

Comment:
We can exclude one of the two routes as they are both rendering the same component.

Reply:
sorry, I should fix conflicts files first. the problem has been fixed.